base/shell/cmd/lang/cs-CZ.rc

```diff
@@ -176,9 +176,9 @@ preset switches by prefixing any switch with - (hyphen)--for example, /-W.\n"
     STRING_DIR_HELP3 " Volume in drive %c has no label.\n"
     STRING_DIR_HELP4 " Volume Serial Number is %04X-%04X\n"
     STRING_DIR_HELP5 "\n     Total Files Listed:\n%16i File(s)% 14s bytes\n"
```

Comment:
STRING_DIR_HELP5 is also affected -> `% 14s` (not in all files though)

Reply:
Fixed.